dask_ml/metrics/classification.py

```diff
@@ -110,7 +122,7 @@ def log_loss(
 
     if y_pred.ndim > 1 and y_true.ndim == 1:
         y_true = y_true.reshape(-1, 1)
-        drop_axis = 1
+        drop_axis: Optional[int] = 1
```

Comment:
Is this optional?

Reply:
Yeah, it gets assingned as `None` on line 129